### ISSUE TITLE
Alarm/ Warning/ Incident documentation page

### ISSUE THREAD

**Issue by Pato-desu** [PauseAI/pauseai-website#250]
It could be in the form of a timeline diagram or not.
Related Discord project: https://discord.com/channels/1100491867675709580/1283432260120612874

**Comment by Pato-desu** [comment]
I'm adding links about warning signs throughout the website right now. 
The important thing about this issue is having examples out there because people learn a looot more from them than statistics/ generalizations made by others.

I'm not sure how useful a dedicated page would be, but I think we should be able to externalize it to https://lethalintelligence.ai/post/category/warning-signs/.I've just asked him if he's thinking of keeping updating it.

**Comment by anthonybailey** [comment]
Fixing link:

https://lethalintelligence.ai/post/category/warning-signs/

The page is... rather histrionic. We want passion and alarm, but not every shit can be completely holy. Also definitely incomplete. Happy enough to link it though.

I think an updated, date-stamped list of linked headlines would be ideal. Don't know if anyone is maintaining one and agree such maintenance would be a huge effort!

Linking our own social media is a possibility.

The "X boats and a helicopter" essays are well chosen and curated. https://thezvi.wordpress.com/2024/12/16/ais-will-increasingly-attempt-shenanigans/#more-24142

**Comment by anthonybailey** [comment]
Oh, top-level.

> I'm adding links about warning signs throughout the website right now.

Always linking to a single page target we control will make our lives easier. Agree it can in turn link or even redirect through elsewhere 

> The important thing about this issue is having examples out there because people learn a looot more from them than statistics/ generalizations made by others.

100%

**Comment by Pato-desu** [comment]
I disagree that it would be a "huge effort" if we had someone who moderately likes to write.
But if the job is almost done elsewhere then I don't see the point of doing it. Also Lethal Intelligence is in our Comms team and if I'm not mistaken he's Michael (the team lead) so I think he would be down to better it.
